projects/packages/backup/src/js/reducers/site-backup-size.js

```diff
@@ -26,6 +29,9 @@ const siteBackupSize = ( state = initialState, action ) => {
 				isFetching: false,
 				loaded: true,
 				size: action.payload.size,
+				minDaysOfBackupsAllowed: action.payload.minDaysOfBackupsAllowed,
```

Comment:
```suggestion
				minDaysOfBackupsAllowed: action.payload?.minDaysOfBackupsAllowed,
```
Nitpick, but I think it is better to use the optional chaining operator `?.` in cases we access a nested property.